test/test_json.py

```diff
@@ -0,0 +1,15 @@
+import json
+import os
+
+HOMEDIR = os.path.expanduser('~')
+file_location = os.path.dirname(os.path.realpath(__file__))
+retriever_root_dir = os.path.abspath(os.path.join(file_location, os.pardir))
```

Comment:
I don't think we need to explicitly track down the root directory here since the tests will always run from that directory. I think we should be able to remove these three lines and just work with `./scripts` below.